server/app/repository/PersistedDurableJobRepository.java

```diff
@@ -1,17 +1,52 @@
 package repository;
 
+import annotations.BindingAnnotations;
+import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
 import io.ebean.DB;
 import io.ebean.Database;
+import java.time.LocalDateTime;
+import java.util.Optional;
+import javax.inject.Inject;
+import javax.inject.Provider;
 import models.PersistedDurableJob;
 
 /** Implements queries related to {@link PersistedDurableJob}. */
 public final class PersistedDurableJobRepository {
 
   private final Database database;
+  private final Provider<LocalDateTime> nowProvider;
 
-  public PersistedDurableJobRepository() {
+  @Inject
+  public PersistedDurableJobRepository(
```

Comment:
**suggestion:** Add javadoc comment here too?

Reply:
We don't usually document c'tors, was there something specific here you'd like documented?

Reply:
Oops yes makes sense!